src/test/java/seedu/address/model/ModelManagerTest.java

```diff
@@ -117,6 +122,11 @@ public void getFilteredStudentList_modifyList_throwsUnsupportedOperationExceptio
         assertThrows(UnsupportedOperationException.class, () -> modelManager.getFilteredStudentList().remove(0));
     }
 
+    @Test
+    public void getMatchedTutorList_modifyList_throwsUnsupportedOperationException() {
+        assertThrows(UnsupportedOperationException.class, () -> modelManager.getMatchedTutorList().remove(0));
```

Comment:
Can use `INDEX_FIRST_PERSON` to reduce magic numbers:
```suggestion
        assertThrows(UnsupportedOperationException.class, () -> modelManager.getMatchedTutorList().remove(INDEX_FIRST_PERSON.getZeroBased()));
```
Same with lines 117 and 122.